bucket/authy.json

```
@@ -0,0 +1,50 @@
+{
+    "version": "1.7.2",
+    "description": "Free desktop app for two factor authentication.",
```

Free is convered by license

```suggestion
    "description": "Two factor authentication client",
```